cpp/libcudf_kafka/tests/CMakeLists.txt

```diff
@@ -27,16 +32,18 @@ function(ConfigureTest test_name)
   target_link_libraries(
     ${test_name} PRIVATE GTest::gmock GTest::gmock_main GTest::gtest_main cudf_kafka
   )
-  add_test(NAME ${test_name} COMMAND ${test_name})
-  install(
-    TARGETS ${test_name}
-    COMPONENT testing
-    DESTINATION bin/gtests/libcudf_kafka
-    EXCLUDE_FROM_ALL
+  rapids_test_add(
+    NAME ${test_name}
+    COMMAND ${test_name}
+    GPUS 1
+    PERCENT 25
```

Comment:
Out of curiosity, how are you picking this number?

Reply:
Entirely abitrary. We currently only have a single `libcudf_kafka` so the percent doesn't matter currently, so I selected a number that felt okay if we introduce more tests in the future.